conf/production.env.example

```diff
@@ -1,72 +1,98 @@
-# Plugins to be loaded at launch
-# CABOT_PLUGINS_ENABLED=cabot_alert_hipchat,cabot_alert_twilio,cabot_alert_email,cabot_alert_slack
-
-DATABASE_URL=postgres://cabot:cabot@localhost:5432/index
-DJANGO_SETTINGS_MODULE=cabot.settings
-LOG_FILE=/dev/null
-PORT=5000
-VENV=/home/ubuntu/venv
-
-# You shouldn't need to change anything above this line
-
+## Cabot UI URL setup
 # Base path to include before generated URLs.  If not defined, uses `/`
 # URL_PREFIX=/
 
-# Local time zone for this installation. Choices can be found here:
+# Used for pointing links back in alerts etc.
+WWW_HTTP_HOST=cabot.example.com
+# Probable values: http, https
+WWW_SCHEME=http
+
+## Local time zone for this installation. Choices can be found here:
 # http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
 TIME_ZONE=Etc/UTC
 
-# Django admin email
+## URL of calendar to synchronise rota with
+CALENDAR_ICAL_URL=http://www.google.com/calendar/ical/example.ics
+
+## Django admin email, 'from' address for email alerts
 ADMIN_EMAIL=you@example.com
 CABOT_FROM_EMAIL=noreply@example.com
 
-# URL of calendar to synchronise rota with
-CALENDAR_ICAL_URL=http://www.google.com/calendar/ical/example.ics
-
-# Django settings
+## Django settings
 CELERY_BROKER_URL=redis://:yourredispassword@localhost:6379/1
-DJANGO_SECRET_KEY=2FL6ORhHwr5eX34pP9mMugnIOd3jzVuT45f7w430Mt5PnEwbcJgma0q8zUXNZ68A
+# Create a random string of mixed case alphanumeric characters, > 40 chars.
+# https://www.browserling.com/tools/random-string is a site that can do this
+DJANGO_SECRET_KEY=CREATE_A_KEY
 
+## Graphite server settings
 # Hostname of your Graphite server instance (including trailing slash)
 GRAPHITE_API=http://graphite.example.com/
 GRAPHITE_USER=username
 GRAPHITE_PASS=password
 
-# From parameter for the graphite request. If not defined, by default take -10 minutes
+# Graphite stats are evaluated through a time period to identify transient
+# failures. This setting determines how many minutes back a test should
+# evaluate.
+# If not defined, evaluate 'now through 10 minutes ago' (-10minute)
 # GRAPHITE_FROM=-10minute
 
-# User-Agent string used for HTTP checks
+## User-Agent string used for Cabot HTTP checks
 HTTP_USER_AGENT=Cabot
 
-# Hipchat integration
-HIPCHAT_ALERT_ROOM=room_name_or_id
-HIPCHAT_API_KEY=your_hipchat_api_key
-
-# Jenkins integration
-JENKINS_API=https://jenkins.example.com/
-JENKINS_USER=username
-JENKINS_PASS=password
+## Email plugin integration
+EMAIL_HOST=smtp.example.com
+# SMTP authentication settings. To disable SMTP authentication, comment out
+#  both EMAIL_USER and EMAIL_PASSWORD.
+EMAIL_USER=smtp_username
+EMAIL_PASSWORD=smtp_password
 
-# SMTP settings
-EMAIL_HOST=smtp.gmail.com
-EMAIL_USER=********@gmail.com
-EMAIL_PASSWORD=*********
+# Typical SMTP port 587 configuration
 EMAIL_PORT=587
 EMAIL_USE_TLS=1
+EMAIL_USE_SSL=1
```

Comment:
TLS and SSL are mutually exclusive - SSL should default to 0 (as e.g. gmail uses TLS)